dev-docs/bidders/pixfuture.md

```diff
@@ -10,12 +10,121 @@ prebid_member: false
 coppa_supported: true
 usp_supported: true
 userIds: criteoId, unifiedId, id5Id, sharedId, identityLink, liveIntentId, fabrickId
-pbs: false
+pbs: true
 pbjs: true
 sidebarType: 1
 ---
-#### Bid Params
 
-| Name                | Scope    | Description                                                                                                                                                                   | Example                                               | Type             |
-|---------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|------------------|
-| `pix_id`       | required | The pix_id is an ID from PixFuture.  The `pix_id` parameter should be a `string`                                                  | `"Abc123"`                                            | `string`         |
+### Overview
+
+The PixFuture Bidder Adapter connects publishers to PixFuture’s demand via Prebid.js and Prebid Server, supporting banner media types. To use this adapter, your PixFuture account must be approved for Prebid integration. Contact <support@pixfuture.com> to request activation or for setup assistance.
+
+#### Solution
+
+To download the PixFuture adapter for Prebid.js, visit [https://docs.prebid.org/download.html](https://docs.prebid.org/download.html) and select the `pixfuture` bidder from the list, along with other bidders you wish to include in your build. For Prebid Server, ensure your server instance includes the `pixfuture` bidder (consult your Prebid Server host for setup details).
+
+**Important:** The `pixfuture` adapter requires account approval from PixFuture. To enable this integration, email <support@pixfuture.com>.
+
+---
+
+### Features
```

Comment:
Please remove this section. It's automatically added to your bidder documentation from the header values above. See https://docs.prebid.org/dev-docs/bidders/pixfuture.html

Reply:
Sorry Bretg, what exactly do I need to remove? Could you provide the code lines numbers from and to?

Reply:
The entire "features section" lines 30-59. It's added automatically by the docs system.

Reply:
fixed